.vscode/settings.json

```diff
@@ -1,4 +1,6 @@
 {
+  "typescript.validate.enable": false,
+  "javascript.validate.enable": false,
```

Comment:
without these my vscode is really confused. happy to remove if this doesn't align with the project vision 😄

Reply:
ah it looks like the following in my user setting is causing this 😄 

>   "typescript.tsserver.experimental.enableProjectDiagnostics": true,

related https://github.com/denoland/vscode_deno/issues/437